charts/argo/values.yaml

```diff
@@ -201,6 +201,12 @@ server:
   # PriorityClass: system-cluster-critical
   priorityClassName: ""
 
+  # Run the argo server in "secure" mode. If this value is set to true, do not
+  # specify "secure" in extraArgs. See the following documentation for more
+  # details on secure mode:
+  # https://argoproj.github.io/argo/tls/#encrypted
+  secure: true
```

Comment:
Should we default to `false` here so that we are not secure by default? and change the comment above to say `If this value is set to false, do not # specify "secure" in extraArgs`

Reply:
Yes. I must have accidentally committed this after testing. It should be false - good catch.

Reply:
I think we don't want to set "--secure" anymore at all in extraArgs because of this value. I will update the comment to say that this replaces the need for setting `--secure` in extraArgs.